docs/src/main/sphinx/connector/kinesis.rst

```diff
@@ -32,23 +32,23 @@ Configuration properties
-``kinesis.aws-region``                  AWS region to be used to read kinesis stream from
-``kinesis.default-schema``              Default schema name for tables
-``kinesis.table-description-location``  Directory containing table description files
-``kinesis.hide-internal-columns``       Controls whether internal columns are part of the table schema or not
-``kinesis.batch-size``                  Maximum number of records to return in one batch
-``kinesis.fetch-attempts``              Read attempts made when no records returned and not caught up
-``kinesis.max-batches``                 Maximum batches to read from Kinesis in one single query
-``kinesis.sleep-time``                  Time for thread to sleep waiting to make next attempt to fetch batch
-``kinesis.iterator-from-timestamp``     Begin iterating from a given timestamp instead of the trim horizon
-``kinesis.iterator-offset-seconds``     Number of seconds before current time to start iterating
-======================================  =======================================================================
+==============================================  ===========================================================================
+Property Name                                       Description
+==============================================  ===========================================================================
+``kinesis.access-key``                              Access key to AWS account or blank to use default provider chain
+``kinesis.secret-key``                              Secret key to AWS account or blank to use default provider chain
+``kinesis.aws-region``                              AWS region to be used to read kinesis stream from
+``kinesis.default-schema``                          Default schema name for tables
+``kinesis.table-description-location``              Directory containing table description files
+``kinesis.table-description-refresh-interval``      How often to get the table description from S3
+``kinesis.hide-internal-columns``                   Controls whether internal columns are part of the table schema or not
+``kinesis.batch-size``                              Maximum number of records to return in one batch
+``kinesis.fetch-attempts``                          Read attempts made when no records returned and not caught up
+``kinesis.max-batches``                             Maximum batches to read from Kinesis in one single query
+``kinesis.sleep-time``                              Time for thread to sleep waiting to make next attempt to fetch batch
+``kinesis.iterator-from-timestamp``                 Begin iterating from a given timestamp instead of the trim horizon
+``kinesis.iterator-offset-seconds``                 Number of seconds before current time to start iterating
+===========================================================================================================================
```

Comment:
Please align like this:
```
==============================================  ===========================================================================
Property Name                                   Description
==============================================  ===========================================================================
``kinesis.access-key``                          Access key to AWS account or blank to use default provider chain
``kinesis.secret-key``                          Secret key to AWS account or blank to use default provider chain
``kinesis.aws-region``                          AWS region to be used to read kinesis stream from
``kinesis.default-schema``                      Default schema name for tables
``kinesis.table-description-location``          Directory containing table description files
``kinesis.table-description-refresh-interval``  How often to get the table description from S3
``kinesis.hide-internal-columns``               Controls whether internal columns are part of the table schema or not
``kinesis.batch-size``                          Maximum number of records to return in one batch
``kinesis.fetch-attempts``                      Read attempts made when no records returned and not caught up
``kinesis.max-batches``                         Maximum batches to read from Kinesis in one single query
``kinesis.sleep-time``                          Time for thread to sleep waiting to make next attempt to fetch batch
``kinesis.iterator-from-timestamp``             Begin iterating from a given timestamp instead of the trim horizon
``kinesis.iterator-offset-seconds``             Number of seconds before current time to start iterating
==============================================  ===========================================================================

```

Could you separate into two commits? 
1. Just make table wider (this commit doesn't contain new property)
2. Add `kinesis.table-description-refresh-interval` property

Reply:
done

Reply:
Please fix footer and commits. There're 3 commits now, but it should be 2 commits.